wled00/ntp.cpp

```diff
@@ -1,10 +1,56 @@
 #include "src/dependencies/timezone/Timezone.h"
 #include "wled.h"
 #include "fcn_declare.h"
+#include <vector>
+
+#ifdef ESP32
+#include "freertos/semphr.h"
+#endif
 
 // WARNING: may cause errors in sunset calculations on ESP8266, see #3400
 // building with `-D WLED_USE_REAL_MATH` will prevent those errors at the expense of flash and RAM
 
+// Dynamic timer storage with thread safety
+std::vector<Timer> timers;
```

Comment:
Use `static` as this vector should not be accessible from elsewhere. You added wrappers.